Selenium_pract_task/Tets_Cases/Validate_that_user_can_search_for_a_position_based_on_criteria.cs

```diff
@@ -5,30 +5,25 @@
 
 namespace Selenium_pract_task.Tets_Cases
 {
-    public class Tests : BaseTest
+    public class Validate_that_user_can_search_for_a_position_based_on_criteria : BaseTest
     {
-        [Test]
-        public void Test1()
+        [TestCase("Java")]
+        [TestCase(".Net")]
+        [TestCase("JavaScript")]
+        public void TestUserPositionSearchBasedOnCriteriaFunctionality(string programinLanguage)
         {
-            //Arrange
-            string programinLanguage;
-
-            //Act
-            programinLanguage = "Java";
-
-            //Assert
             driver.Navigate().GoToUrl("https://www.epam.com/");
             var epamMainPageContext = new EpamMainPageContext(driver);
-            epamMainPageContext.verifyCookiesHendler()
-                .clickOnCareersLink()
+            epamMainPageContext.VerifyCookiesHandler()
+                .ClickOnCareersLink()
                 .ScrolePage<EpamCareersPageContext>(8)
```

Comment:
Probably, let`s try to use:
IJavaScriptExecutor js = (IJavaScriptExecutor)driver;
js.ExecuteScript("arguments[0].scrollIntoView();", epamCereersPage.CheckBoxRemote);